integrations/git_test.go

```diff
@@ -197,10 +212,17 @@ func testGit(t *testing.T, u *url.URL) {
 					t.Run("Little", func(t *testing.T) {
 						PrintCurrentTest(t)
 						littleLFS = commitAndPush(t, littleSize, dstPath)
+						lockFileTest(t, littleLFS, dstPath)
+
 					})
 					t.Run("Big", func(t *testing.T) {
+						if testing.Short() {
+							return
```

Comment:
Please use t.Skip method to indicate we skipped the test.
        
```suggestion
							t.Skip("skipping test in short mode.")
```

Reply:
Done

Reply:
In fact this was not ...